CMakeLists.txt

```diff
@@ -1386,21 +1397,32 @@ if(BUILD_TESTS)
       PROPERTIES FAIL_REGULAR_EXPRESSION "FAILED" TIMEOUT 1000)
   endfunction(configure_test)
 
+  # Installed as part of the installable testsuite.
+  install(TARGETS test_lib
+          LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}/rr)
+  install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/bin/test-monitor
+                   ${CMAKE_CURRENT_BINARY_DIR}/bin/cpuid
+          DESTINATION ${CMAKE_INSTALL_LIBDIR}/rr/testsuite/obj/bin)
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/CTestTestfile.cmake
+          DESTINATION ${CMAKE_INSTALL_LIBDIR}/rr/testsuite/obj/)
+
+  SET(TESTSUITE_BINARY_DIR ".")
+
   foreach(test ${BASIC_TESTS} ${BASIC_CPP_TESTS} ${OTHER_TESTS})
     add_test(${test}
-      bash ${CMAKE_SOURCE_DIR}/src/test/basic_test.run ${test} -b ${PROJECT_BINARY_DIR} ${TEST_MONITOR_DEFAULT_TIMEOUT})
+      bash "../rr/src/test/basic_test.run" ${test} -b ${TESTSUITE_BINARY_DIR} ${TEST_MONITOR_DEFAULT_TIMEOUT})
```

Comment:
This assumes that the name of a source checkout is `rr`. That's not necessarily the case. E.g. mine is called `rr2` and now the tests don't work anymore.

Reply:
I will fix this. Just need to replace 'rr' with the basename of the source directory.